config/config.go

```diff
@@ -91,22 +34,35 @@ var allowedDBLogModes = map[string]bool{
 	"silent":   true,
 }
 
-// New : Construct a new config variable
-func New() *Config {
-	cfg := &Config{}
-	*cfg = Defaults
-	return cfg
+// Construct a new config variable
+func init() {
+	getDefaultConfig()
+	privateConf = *DefaultConfig
+	Conf = &privateConf
+	overrideDefaults()
+}
+
+func overrideDefaults() {
+	path := "config/config.yml"
```

Comment:
This'll cause issues in every unit test that imports this package. (Except for the main package,)

Unit tests don't use the project root as the working directory and will fail to find these files.
(The init function is run before the test functions)

You'll have to make sure the default config file is always found or the tests will fail in strange ways.